app/src/components/Scheduling/Header.js

```diff
@@ -30,9 +18,7 @@ export default function Header({ textOptions }) {
           width: '200px',
           textAlign: 'center',
```

Comment:
Line 26 of this component is commented out. If it's not being used, please remove it.